docs/development/local-development.md

```diff
@@ -46,17 +46,27 @@ If pushing to an external image registry, you may need to provide credentials to
 
 ```bash
 $ make ko
-$ ko login <IMAGE_REGISTRY> -u <USERNAME> -p <PASSWORD>
+$ bin/ko login <IMAGE_REGISTRY> -u <USERNAME> -p <PASSWORD>
 ```
 
-Next, build the operator image as specified above, and push to a container registry that can be accessed by the cluster:
+Next, use the ko-deploy make target to deploy to Kubernetes:
 
 ```bash
-$ make build IMAGE_REPO="<IMAGE_REGISTRY>/<USERNAME>" TAG="<TAG>"
+$ make ko-deploy IMAGE_REPO="<IMAGE_REGISTRY>/<USERNAME>" TAG="<TAG>"
 ```
 
-Finally, use the `make deploy` command with appropriate `IMAGE_REPO` and `TAG` arguments to deploy to the cluster.
+If deploying to a local KinD cluster, use `IMAGE_REPO=kind.local` and `TAG=latest`.
 
-```bash
-$ make deploy IMAGE_REPO="<IMAGE_REGISTRY>/<USERNAME>" TAG="<TAG>"
-```
+*Note: this will result in changes to `kustomization.yaml` files.*
+*Do not check these changes into git unless you intend to change how the operator is bundled for OLM.*
```

Comment:
This seems like a footgun waiting to go off. Is there any way to make it harder to misuse? Maybe kustomize into a temp file and apply that? Or just remove kustomize from `ko-deploy` entirely?

Reply:
Since I can see folks deploying to kind or docker itself, kustomize into a temp file feels like the right approach.

Reply:
Given the way that kustomize works, I don't think we can go with this approach without copying the config directory to a tmpdir and running kustomize there.

The alternative is to default kustomize to the `ko://` image. We still have the footgun problem because `make deploy` and `make bundle` edit the kustomization config.